app/src/main/res/layout/item_row.xml

```diff
@@ -1,57 +1,65 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
@coder2699 Why changes are made in this file?

Reply:
@rob729 To add the Pause/Resume Switch